src/helpers/git.js

```diff
@@ -31,14 +31,18 @@ module.exports = new (class Git {
       }
     }
 
-    // Set config
-    this.config('user.name', gitUserName)
-    this.config('user.email', gitUserEmail)
-
-    // Update the origin
-    if (githubToken) {
-      this.updateOrigin(`https://x-access-token:${githubToken}@${gitUrl}/${GITHUB_REPOSITORY}.git`)
-    }
+    // use a self-invoking async function to await promises inside a constructor
+    // this avoids git config lock errors which are triggered when multiple `git config` commands are run
+    (async () => {
```

Comment:
Maybe an idea to move this to something like a `init` and call `await git.init()` somewhere [here](https://github.com/TriPSs/conventional-changelog-action/blob/releases/v3/src/index.js#L81), then we are sure these are set correctly before being used.